docs/themes/hugo-docs/layouts/_default/article.html

```diff
@@ -0,0 +1,23 @@
+{{ $currentVersion := getenv "CURRENT_VERSION" }}
+{{ $versionString := getenv "VERSIONS" }}
+{{ $versions := split $versionString "," }}
+{{ $latestVersion := index $versions 0 }}
+
+{{ if (eq $currentVersion "master") }}
+  <div class="alert alert-warning">
+    <i class="fa fa-warning"></i> You are looking at the docs for the unreleased <code>master</code> branch of Dgraph. The latest version is <a href="{{ .Site.BaseURL }}/..">{{ $latestVersion }}</a>.
+  </div>
+{{ else if not (eq $latestVersion $currentVersion) }}
+  <div class="alert alert-warning">
+    <i class="fa fa-warning"></i> You are looking at the docs for an older version of Dgraph ({{ $currentVersion }}). The latest version is <a href="{{ .Site.BaseURL }}/..">{{ $latestVersion }}</a>.
```

Comment:
This looks out of place. Why is there a mention of dgraph docs on the badger docs page?